x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -49,12 +55,13 @@ import { useRefresh } from '../routing/use_refresh';
 import { SavedObjectsWarning } from '../components/saved_objects_warning';
 import { TestTrainedModelFlyout } from './test_models';
 
-type Stats = Omit<TrainedModelStat, 'model_id'>;
+type Stats = Omit<TrainedModelStat, 'model_id' | 'deployment_stats'>;
 
 export type ModelItem = TrainedModelConfigResponse & {
   type?: string[];
-  stats?: Stats;
+  stats?: Stats & { deployment_stats: TrainedModelDeploymentStatsResponse[] };
   pipelines?: ModelPipelines['pipelines'] | null;
```

Comment:
As discussed offline, currently we are only checking for model IDs and deployment IDs for associated pipelines

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/7405507/234259815-2d9ffd54-153e-454a-9004-b4c55e6d03fe.png">

Reply:
Added in https://github.com/elastic/kibana/pull/155375/commits/27fb400bf94c4897d834b3df9a3ff7bc90ce45d1